### PR TITLE
USHIFT-4278: Specify default value for .storage.optionalCsiComponents that is an accepted value

### PR DESCRIPTION
**Pull request by copejon**

### CODE REVIEW

File: packaging/microshift/config.yaml [hunk 1]
```diff
@@ -131,6 +131,9 @@ storage:
     # storage driver. Like CSIStorageDriver, uninstallation is not supported as this can lead to orphaned storage
     # objects.
     # Allowed values are: unset, [], or one or more of ["snapshot-controller", "snapshot-webhook"]
+    # example:
+    #   - snapshot-controller
+    # - snapshot-webhook
```
Comment (pmtk) [inline packaging/microshift/config.yaml:136]: Is this auto-generated?
```suggestion
    # Example:
    # - snapshot-controller
    # - snapshot-webhook
```

Comment (copejon) [reply packaging/microshift/config.yaml:136]: I know, controller-gen _insists_ on generating it this way. I tried a handful of permutations on the tag's value but it always comes out this way.

Comment (pmtk) [reply packaging/microshift/config.yaml]: `ci/prow/verify` failed, I don't think we should include this "suggestions"

**Comment by copejon** [comment]
@pmtk I reverted the comment fix. The format was ping as an error by the verify script 🤷🏻

**Review by pacevedom — APPROVED** [review]
/lgtm

**Pull request merged**